packages/react-card/src/components/Card/useCardStyles.ts

```diff
@@ -21,7 +24,7 @@ const useStyles = makeStyles({
     gap: '12px',
     borderRadius: theme.borderRadiusMedium,
 
-    '> .fluentui-react-card-preview': {
+    [`> .${cardPreviewClassName}`]: {
```

Comment:
I updated selector to use the constant, classnames in `CardPreview` were also updated.